setupext.py

```diff
@@ -151,7 +151,8 @@ def has_include_file(include_dirs, filename):
     directories in `include_dirs`.
     """
     if sys.platform == 'win32':
-        include_dirs += os.environ.get('INCLUDE', '.').split(';')
+        include_dirs = list(include_dirs)
+        include_dirs += os.environ.get('INCLUDE', '.').split(os.pathsep)
```

Comment:
probably worth clarifying in a comment?  I would have missed the point of the change if not for the commit message...

Reply:
I'm also concerned about the change from `split(';')` to `split(os.pathsep)`. Those are not the same, but there's nothing indicating the rationale behind them.

Reply:
@dopplershift `os.pathsep` == `;` on windows https://docs.python.org/2/library/os.html#os.pathsep
```
Visual C++ 2015 x64 Native Build Tools Command Prompt
C:\Program Files (x86)\Microsoft Visual C++ Build Tools>set INCLUDE
INCLUDE=C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE;C:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\ucrt;C:\Program Files (x86)\Windows Kits\NETFXSDK\4.6\include\um;C:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\shared;C:\Program Files(x86)\Windows Kits\10\include\10.0.15063.0\um;C:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\winrt;
```

Reply:
Oops. 🐑 Thanks for clarifying.